flutter_appauth/android/src/main/java/io/crossingthestreams/flutterappauth/FlutterAppauthPlugin.java

```diff
@@ -431,7 +432,9 @@ private void performEndSessionRequest(AuthorizationServiceConfiguration serviceC
     }
 
     private void finishWithTokenError(AuthorizationException ex) {
-        finishWithError(TOKEN_ERROR_CODE, String.format(TOKEN_ERROR_MESSAGE_FORMAT, ex.error, ex.errorDescription));
+        final Throwable cause = ex.getCause();
+        final String causeDescription = cause != null ? cause.getMessage() : null;
+        finishWithError(TOKEN_ERROR_CODE, String.format(TOKEN_ERROR_MESSAGE_FORMAT, ex.errorDescription, causeDescription));
```

Comment:
Dropping the use of the `error` property of the `AuthorizationException` could lead to less information about what error happened in other scenarios. if you look at the API docs for this property (https://github.com/openid/AppAuth-Android/blob/ceb253b8118e481f3fe6f648772a8d16179a11fe/library/java/net/openid/appauth/AuthorizationException.java#L620), it's meant for errors that are part of OAuth 2 protocol. There's also code within AppAuth Android SDK that populates it https://github.com/openid/AppAuth-Android/blob/ceb253b8118e481f3fe6f648772a8d16179a11fe/library/java/net/openid/appauth/AuthorizationService.java#L655

With this in mind, I would say it's better to cater for all scenarios by first reverting the code to what it was before so that the `error` is still returned as it did before and pass the `causeDescription` to the third parameter of the method call at https://github.com/MaikuB/flutter_appauth/blob/49d7919e53fba8a7372884485b12f2849089b502/flutter_appauth/android/src/main/java/io/crossingthestreams/flutterappauth/FlutterAppauthPlugin.java#L447 so that it's populates the [details](https://api.flutter.dev/flutter/services/PlatformException/details.html) property of the `PlatformException` on the Flutter side